### ISSUE TITLE
HexEditDialog's  unnecessary taborders 

### ISSUE THREAD

**Issue by stdust** [x64dbg/x64dbg#3237]
### Feature type

Other

### Describe the feature

HexEditDialog's tab orders have been assigned to code for less frequently used codepages and to keep size, etc
 so unnecessary tab orders should be deleted

I propose the following taborder

![HexEditDialog11](https://github.com/x64dbg/x64dbg/assets/139965217/3a59aa24-44f6-4e08-a0e2-032c73455e3a)
![HexEditDialog22](https://github.com/x64dbg/x64dbg/assets/139965217/3815639e-4ee2-4c07-80e6-a809bf17a0e9)
![HexEditDialog33](https://github.com/x64dbg/x64dbg/assets/139965217/90b099c9-764f-439b-83b2-33bd12dea3d2)
